# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.cpp
# Commit: 99e71b905661
# Full Hash: 99e71b9056618a081f8bc2fb7803ba208db3a5b0
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-10 09:45:41
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin,devtools-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 68f119112940 -r 99e71b905661 netwerk/protocol/http/nsHttpChannel.cpp
--- a/netwerk/protocol/http/nsHttpChannel.cpp	Tue Apr 09 15:26:45 2024 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.cpp	Tue Apr 09 15:26:46 2024 +0000
@@ -7690,6 +7690,7 @@
       return "proxy"_ns;
     }
     if (contentType.EqualsLiteral(APPLICATION_BROTLI) ||
+        contentType.EqualsLiteral(APPLICATION_ZSTD) ||
         contentType.Find("zip") != kNotFound ||
         contentType.Find("compress") != kNotFound) {
       return "compressed"_ns;